server/events.go

```diff
@@ -776,10 +797,10 @@ func (s *Server) sendAccConnsUpdate(a *Account, subj string) {
 	a.mu.RLock()
 
 	// If no limits set, don't update, no need to.
-	if a.mconns == jwt.NoLimit && a.mleafs == jwt.NoLimit {
-		a.mu.RUnlock()
-		return
-	}
+	//if a.mconns == jwt.NoLimit && a.mleafs == jwt.NoLimit {
```

Comment:
Not sure about that...

Reply:
Meaning remove the commented out code or tracking account connections with no limit set?

Reply:
I meant remove the code if it was the intent.